repository-data/application/src/main/resources/hcm-config/namespaces/hee/news.yaml

```diff
@@ -0,0 +1,262 @@
+definitions:
+  config:
+    /hippo:namespaces/hee/news:
+      jcr:primaryType: hipposysedit:templatetype
+      jcr:mixinTypes: ['editor:editable', 'mix:referenceable']
+      jcr:uuid: 3ee9cb76-9bac-4c0d-9b6a-e8f2d1349414
+      /hipposysedit:nodetype:
+        jcr:primaryType: hippo:handle
+        jcr:mixinTypes: ['mix:referenceable']
+        jcr:uuid: 2f8dbd3d-bee5-4659-be9d-87e88e518ef4
+        /hipposysedit:nodetype:
+          jcr:primaryType: hipposysedit:nodetype
+          jcr:mixinTypes: ['mix:referenceable', 'hipposysedit:remodel']
+          jcr:uuid: 5d865ca8-9d18-464c-8bc8-9ac82aa47791
+          hipposysedit:node: true
+          hipposysedit:supertype: ['hee:basedocument', 'hippostd:relaxed', 'hippotranslation:translated']
+          hipposysedit:uri: http://www.heecmsplatform.com/hee/nt/1.0
+          /title:
+            jcr:primaryType: hipposysedit:field
+            hipposysedit:mandatory: false
+            hipposysedit:multiple: false
+            hipposysedit:ordered: false
+            hipposysedit:path: hee:title
+            hipposysedit:primary: false
+            hipposysedit:type: String
+            hipposysedit:validators: [required]
+          /summary:
+            jcr:primaryType: hipposysedit:field
+            hipposysedit:mandatory: false
+            hipposysedit:multiple: false
+            hipposysedit:ordered: false
+            hipposysedit:path: hee:summary
+            hipposysedit:primary: false
+            hipposysedit:type: Text
+            hipposysedit:validators: [optional]
+          /contentBlocks:
+            jcr:primaryType: hipposysedit:field
+            hipposysedit:multiple: true
+            hipposysedit:ordered: true
+            hipposysedit:path: hee:contentBlocks
+            hipposysedit:type: hippo:compound
+            hipposysedit:validators: [contentblocks-validator]
+          /relatedContent:
```

Comment:
`Blog post` DocumentType currently doesn't seem to be supporting `Related content`. Since, `News` DocumentType is essentially based on `Blog post`, just wondering if we need to support `Related content` for `News` ? Couldn't see a mention of this on `NWPS-796` & `NWPS-770` tickets. Could we check with Eleanor/Will on this ?

If this is required, then `repository-data/webfiles/src/main/resources/site/freemarker/hee/catalog/news-main.ftl` template should be updated to support this.

Reply:
Thanks. Confirmed in ticket to keep for now. Have updated the ftl.